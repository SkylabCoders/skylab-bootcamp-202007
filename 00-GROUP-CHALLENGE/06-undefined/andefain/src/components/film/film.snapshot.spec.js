import renderer from 'react-test-renderer';
import React from 'react';
import Film from './film';
import { BrowserRouter as Router } from 'react-router-dom';

describe('Film snapshot', () => {
	const filmTree = renderer.create(
		<Router>
			<Film />
		</Router>
	);
	it('Should match', () => {
		expect(filmTree.toJSON()).toMatchSnapshot();
	});
});
