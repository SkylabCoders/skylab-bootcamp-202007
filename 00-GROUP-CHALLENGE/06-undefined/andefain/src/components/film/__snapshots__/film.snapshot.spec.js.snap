// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Film snapshot Should match 1`] = `
<section
  className="film-container"
>
  <div
    className="trailer-container"
  >
    <a
      href="/film"
      onClick={[Function]}
    >
      <img
        alt="Poster"
        className="poster-img"
      />
    </a>
    <a
      href="/film"
      onClick={[Function]}
    >
      <img
        alt="Trailer"
        className="trailer-img"
        src=""
      />
    </a>
  </div>
  <div
    className="details-container"
  >
    <div
      className="details-header"
    >
      <h1
        className="film-details__title"
      />
      <img
        alt="Start icon"
        className="icon-like-desktop"
        height="30px"
        onClick={[Function]}
        src="https://trello-attachments.s3.amazonaws.com/5f294480df57d910f5d84ab9/512x512/5f83a4529179eef86fac458fd103c413/favorito.png"
        width="30px"
      />
    </div>
    <span />
    <div
      className="film-details__head"
    >
      <span>
        <span />
        <span>
           | 
        </span>
        <span />
        <span>
           | 
        </span>
        <span />
      </span>
    </div>
    <div
      className="film-details__plot"
    >
      <h3
        className="plot-title"
      >
        Plot
      </h3>
      <p
        className="plot-details"
      />
    </div>
  </div>
</section>
`;
